package.json

```diff
@@ -26,7 +27,7 @@
         "@babel/polyfill": "7.0.0",
         "@babel/preset-env": "^7.1.0",
         "@babel/preset-react": "7.0.0",
-        "@babel/runtime": "^7.5.0",
+        "@babel/runtime": "7.0.0-beta.42",
```

Comment:
Why did we revert to the previous version of babel runtime??

Reply:
I was seeing how wave.js works and ties into the materials-designer. I was trying to debug an error I was getting when i tried to run wave.js linked to a local changes using `file:/stack/lib/wave`. I will revert back to `"^7.5.0"`